src/ui/mathliveview.js

```diff
@@ -0,0 +1,35 @@
+import InputView from '@ckeditor/ckeditor5-ui/src/view';
+import { Template } from '@ckeditor/ckeditor5-ui';
+
+export default class MathLiveView extends InputView {
+	constructor( locale, { options, attributes, style } ) {
+		super( locale );
+		this._options = options;
+		const bind = Template.bind( this, this );
```

Comment:
https://ckeditor.com/docs/ckeditor5/latest/framework/guides/architecture/ui-library.html#definition uses `const bind = this.bindTemplate;`. I don't know if there's a practical difference but docs-way feels safer.